talks/karen-haryan.md

```diff
@@ -25,8 +25,8 @@ Including sharing on social media such as Meetup.com, Twtter, Linkedin, etc.
 
 **Abstract:**
 Story of how a Student caused entire Company to go down. But damn We have heard such stories a lot. So let's talk about solutions.
-We haven't watched Arnold Schwarzenegger speeches about success, we went agains all best brains books, tutorials, and suggestions.
-Presentation will be more focused to share how company culture is important when board memerbs are taking decisions for such situations.
+We haven't watched Arnold Schwarzenegger speeches about success, we went against all best brains books, tutorials, and suggestions.
```

Comment:
What's a brains book? :)

Reply:
there are books starting with best brains and education institutes BestBrains ;)